charts/cass-operator/crds/customresourcedefinition.yaml

```diff
@@ -4819,68 +4819,68 @@ spec:
-                                  is the key and content is the value. If specified,
-                                  the listed keys will be projected into the specified
-                                  paths, and unlisted keys will not be present. If
-                                  a key is specified which is not present in the ConfigMap,
-                                  the volume setup will error unless it is marked
-                                  optional. Paths must be relative and may not contain
-                                  the '..' path or start with '..'.
-                                items:
-                                  description: Maps a string key to a path within
-                                    a volume.
-                                  properties:
-                                    key:
-                                      description: The key to project.
-                                      type: string
-                                    mode:
-                                      description: 'Optional: mode bits to use on
-                                        this file, must be a value between 0 and 0777.
-                                        If not specified, the volume defaultMode will
-                                        be used. This might be in conflict with other
-                                        options that affect the file mode, like fsGroup,
-                                        and the result can be other mode bits set.'
-                                      format: int32
-                                      type: integer
-                                    path:
-                                      description: The relative path of the file to
-                                        map the key to. May not be an absolute path.
-                                        May not contain the path element '..'. May
-                                        not start with the string '..'.
-                                      type: string
-                                  required:
-                                    - key
-                                    - path
-                                  type: object
-                                type: array
-                              name:
-                                description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
-                                  TODO: Add other useful fields. apiVersion, kind,
-                                  uid?'
-                                type: string
-                              optional:
-                                description: Specify whether the ConfigMap or its
-                                  keys must be defined
-                                type: boolean
-                            type: object
+#                          configMap:
```

Comment:
What are we accomplishing by commenting this out?

Reply:
I don't remember doing this. I think this came in off a rebase and needs to be adjusted.

Reply:
It looks like the latest rebase off of main removed this change. Resolving the comment here.